processing/src/main/java/io/druid/segment/data/ColumnarDoubles.java

```diff
@@ -38,7 +39,7 @@
   @Override
   void close();
 
-  default ColumnValueSelector<Double> makeColumnValueSelector(ReadableOffset offset)
+  default ColumnValueSelector<Double> makeColumnValueSelector(ReadableOffset offset, ImmutableBitmap nullValueBitmap)
```

Comment:
It should check if the provided bitmap is empty, and return optimized impl then with `isNull() { return false; }`

Reply:
added a check for empty bitmap.

Reply:
Same as in https://github.com/druid-io/druid/pull/5278#discussion_r166073102

Reply:
done.